doc/0000-ccl_note/main.tex

```diff
@@ -511,11 +511,11 @@ \subsubsection{Spline parameters \& the INI file}
 \subsubsection{Extrapolation for the nonlinear power spectrum}
 \label{sec:NLextrapol}
 
-The computation of the power spectrum from {\tt CLASS} can be significantly sped up by extrapolating in the range $k>$~{\tt K$\_$MAX$\_$SPLINE} and $k<$~{\tt K$\_$MIN}. In this section, we describe the implementation of the extrapolation and the accuracy attained. These tests are performed in a flat $\Lambda$CDM cosmology with $\Omega_c=0.25$, $\Omega_b=0.05$, $A_s=2.1\times10^{-9}$, $h=0.7$ and $n_s=0.96$.
+The computation of the power spectrum from {\tt CLASS} can be significantly sped up by extrapolating in the range $k>$~{\tt K$\_$MAX$\_$SPLINE} and $k<$~{\tt K$\_$MIN$\_$SPLINE}. In this section, we describe the implementation of the extrapolation and the accuracy attained. These tests are performed in a flat $\Lambda$CDM cosmology with $\Omega_c=0.25$, $\Omega_b=0.05$, $A_s=2.1\times10^{-9}$, $h=0.7$ and $n_s=0.96$.
```

Comment:
Is this K_MIN_DEFAULT or K_MIN_SPLINE? Or are we referring here to a different number?

Reply:
We discussed this in the telecon - these are different parameters. One sets the minimum K for integration, the other, for splining. K_MIN_SPLINE is not directly available to the user.